kyo-core/shared/src/main/scala/kyo/Async.scala

```diff
@@ -248,6 +248,47 @@ object Async extends AsyncPlatformSpecific:
     ): A < (Abort[E] & Async & S) =
         race[E, A, S](first +: rest)
 
+    /** Races multiple computations and returns the result of the first to complete. When one computation completes, all other computations
+      * are interrupted.
+      *
+      * WARNING: Executes all computations concurrently without bounds. Use with caution on large sequences to avoid resource exhaustion. On
+      * platforms supporting parallel execution (like JVM), computations may run in parallel.
+      *
+      * @param seq
+      *   The sequence of computations to race
+      * @return
+      *   The result of the first computation to complete
+      */
+    def raceFirst[E, A, S](
```

Comment:
What do you think of `race` => `raceSuccess` and `raceFirst` => `race`?

Reply:
Note: https://zio.dev/overview/basic-concurrency/#racing
I am more used to `race` is the first success. Another way to solve it, use `raceSuccess`/`raceFirst`.